### PR TITLE
standardize imports

### PR DESCRIPTION
**Pull request by dhrubabasu**
## Why this should be merged

Makes our imports consistent 🧹 

## How this works

find + replace

## How this was tested

CI

### CODE REVIEW

**Review by ceyonur — APPROVED** [review]
My fav PR type: deleted > added
LGTM

**Review by abi87 — APPROVED** [review]

**Comment by darioush** [comment]
Could you add a linter that enforces this?

File: scripts/lint.sh [hunk 1]
```diff
@@ -51,6 +51,12 @@ function test_license_header {
   "${files[@]}"
 }
 
+function test_single_import {
+  if grep -R -zo -e 'import (\n\t".*"\n)' .; then
+    return 1
```
Comment (richardpringle) [inline scripts/lint.sh:56]: Should also add a log to say why the linter failed.

Comment (dhrubabasu) [reply scripts/lint.sh:56]: It does print it:
```
START: 'single_import' at Thu May  4 17:37:48 EDT 2023
./snow/consensus/snowball/binary_slush.go:import (
        "fmt"
)
./snow/consensus/snowball/binary_snowflake.go:import (
        "fmt"
)
./snow/consensus/snowball/binary_snowball.go:import (
        "fmt"
)
FAIL: 'single_import' failed at Thu May  4 17:37:50 EDT 2023
```

**Review by darioush — APPROVED** [review]

**Review by StephenButtolph — APPROVED** [review]

**Pull request merged**